Neos.Flow/Classes/Command/RoutingCommandController.php

```diff
@@ -73,8 +66,9 @@ public function listCommand(): void
     {
         $this->outputLine('<b>Currently registered routes:</b>');
         $rows = [];
+        $routes = $this->routesProvider->getRoutes();
         /** @var Route $route */
-        foreach ($this->router->getRoutes() as $index => $route) {
+        foreach ($routes as $index => $route) {
```

Comment:
Why a separate variable?
This could be `foreach ($this->routesProvider->getRoutes() as $index => $route)`

This will spare one assigned variable. Besides the `@var` annotation should be obsolete now if `Routes` is typed correctly to return `array<int, Route>`